src/python_testing/TC_RVCOPSTATE_2_1.py

```diff
@@ -94,15 +94,16 @@ async def test_TC_RVCOPSTATE_2_1(self):
                 asserts.fail("The --app-pid flag must be set when PICS_SDK_CI_ONLY is set")
```

Comment:
Can't comment up there, but having the endpoint be set is actually pretty important for this test because endpoint == None defaults to EP0, which is not what we want here. Can you add an assertion below that line that endpoint is actually set?

Reply:
Hi Cecille, 
I have added an additional check in to make sure the endpoint was not None or 0, please let me know if this will resolve the issue noticed and mentioned in your above comment!